utils/llm.py

```diff
@@ -31,7 +31,10 @@ def model_from_alias(model: str):
 
 def get_max_prompt_size(model: str):
     model = model_from_alias(model)
-    return litellm.model_cost[model]["max_input_tokens"]
+    model_info = litellm.model_cost.get(model, None)
+    if model_info:
+        return model_info["max_input_tokens"]
+    return 0
```

Comment:
I'm not sure about this `return 0`. Maybe an error would be better?

Reply:
It is to stop the error! Litellm doesn't have all possible models in its repository. So it breaks when trying to retireve tokens. In this case, we should just trust the user.